region/crateria/central/Parlor and Alcatraz.json

```diff
@@ -3042,6 +3127,34 @@
       "unlocksDoors": [{"types": ["ammo"], "requires": []}],
       "flashSuitChecked": true
     },
+    {
+      "link": [8, 8],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "requires": [
+        "h_ZebesIsAwake",
+        {"not": "f_ZebesSetAblaze"},
+        {"obstaclesCleared": ["R-Mode"]},
+        {"or": [
+          "h_CrystalFlash",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 100}}
+          ]}
+        ]},
+        {"canShineCharge": {"usedTiles": 25, "steepUpTiles": 3, "steepDownTiles": 3, "openEnd": 0}},
+        {"autoReserveTrigger": {}},
+        "canRModeSparkInterrupt"
+      ],
+      "resetsObstacles": ["R-Mode"],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Farm Geemers and Skree for energy or else Crystal Flash.",
+        "The bomb wall respawns so Crystal Flash can't be used to clear it.",
+        "Shinecharge and get interrupted by the Ripper by the save room door."
```

Comment:
Using the Ripper would require `canShinechargeMovementTricky`.  They should be able to leave a Geemer alive if that isn't on.